### PR TITLE
feat: add api to get a transaction from the wallet

### PR DESCRIPTION
**Pull request by pedroferreira1**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -225,6 +225,21 @@ walletRouter.get('/tx-history', (req, res) => {
   }
 });
 
+/**
+ * GET request to get a transaction from the wallet
+ * For the docs, see api-docs.js
+ */
+walletRouter.get('/transaction', (req, res) => {
```
Comment (msbrogli) [inline index.js:232]: Missing sanitization.

File: index.js [hunk 2]
```diff
@@ -225,6 +225,21 @@ walletRouter.get('/tx-history', (req, res) => {
   }
 });
 
+/**
+ * GET request to get a transaction from the wallet
+ * For the docs, see api-docs.js
```
Comment (msbrogli) [inline index.js:230]: Missing update in the api-docs.js.

**Review by msbrogli — CHANGES_REQUESTED** [review]

File: index.js (hunk 2 above)
Comment (pedroferreira1) [reply index.js:230]: Done

File: index.js (hunk 1 above)
Comment (pedroferreira1) [reply index.js]: Done!

**Review by msbrogli — APPROVED** [review]

**Pull request merged**